keyboards/s_ol/0xc_pad/info.json

```diff
@@ -0,0 +1,33 @@
+{
+    "keyboard_name": "0xC.pad",
+    "url": "https://s-ol.nu/0xC.pad",
+    "debounce": 5,
+    "diode_direction": "ROW2COL",
+    "matrix_pins": {
+      "rows": ["B7", "B6", "B5", "B4"],
+      "cols": ["D4", "D3", "D2", "D1"]
+    },
+    "layouts": {
+        "LAYOUT": {
+            "layout": [
+                {"x":1, "y":0},
+                {"x":2, "y":0},
+                {"x":0.5, "y":1},
+                {"x":1.5, "y":1},
+                {"x":2.5, "y":1},
+                {"x":0, "y":2},
+                {"x":1, "y":2},
+                {"x":2, "y":2},
+                {"x":3, "y":2},
+                {"x":0.5, "y":3},
+                {"x":1.5, "y":3},
+                {"x":2.5, "y":3}
+            ]
+        }
+    },
+    "usb": {
+      "vid": "0x732D",
+      "pid": "0x0C70",
+      "device_ver": "0.0.1"
```

Comment:
```suggestion
      "device_version": "0.0.1"
```

Reply:
ah okay, I guess lib/python/qmk/info.py:77 should also be adjusted at some point then?

Reply:
https://github.com/qmk/qmk_firmware/pull/14580

Reply:
I see 👍 Change is applied.